docs/src/experimental/intro.md

```diff
@@ -12,6 +12,13 @@ The code in `experimental` is supposed to be mathematically correct,
 be stabilized. Also code is allowed to reside in `experimental` while it is
 brought up to Oscar standard.
 
+!!! danger "Dependencies"
+    - Code from `src` must never use code from `experimental`
+    - Say there are two packages `A` and `B` in `experimental`, and `B` depends
+      on `A`. That means that `B` cannot be moved to `src` before `A`. Worse:
+      If `A` gets abandoned, `B` might share that fate. So please consider
+      carefully in such situations.
```

Comment:
Please note, that this currently is the case. Cf. [binomial_ideals](https://github.com/oscar-system/Oscar.jl/blob/22a59d64709d4ba0ddb01824daf78c8dc7c86671/src/Oscar.jl#L515-L516)

Reply:
That is not the only case. Nevertheless we want to avoid this in the future, since now the intention of `experimental` has become clearer. But as discussed in #2146 I will leave older packages alone for the time being.